compiler/hie-core/test/bazel/Development/IDE/Test/Runfiles.hs

```diff
@@ -0,0 +1,16 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Development.IDE.Test.Runfiles
+  ( locateHieCoreExecutable
+  ) where
+
+import System.FilePath ((</>), FilePath)
+
+import DA.Bazel.Runfiles
+
+
+locateHieCoreExecutable :: IO FilePath
+locateHieCoreExecutable = locateRunfiles hieCoreExePath
+  where
+    hieCoreExePath = mainWorkspace </> exe "compiler/hie-core/hie-core-exe"
```

Comment:
I was about to suggest to put this in a single file and switch with a CPP define until I remembered that we spent over a day fighting CPP :grin: